$NetBSD: patch-configure.ac,v 1.3 2021/02/06 06:23:22 ryoon Exp $

* check for pkgsrc libixion, used by math/py-libixion

* Improve POSIX shell portability

--- configure.ac.orig	2019-08-08 20:56:25.000000000 +0000
+++ configure.ac
@@ -142,6 +142,7 @@ AS_IF([test "x$enable_log_trace" != "xno
 AS_IF([test "x$enable_python" != "xno"], [
     AM_PATH_PYTHON(3)
     PKG_CHECK_MODULES([PYTHON], [python3 >= 3.4])
+    PKG_CHECK_MODULES([DEPLIBIXION], [libixion-$IXION_API_VERSION >= $IXION_API_VERSION])
 ])
 
 AS_IF([test "x$enable_threads" != "xno"], [
